src/main/scala-3/com/sksamuel/scapegoat/inspections/AvoidRequire.scala

```diff
@@ -0,0 +1,39 @@
+package com.sksamuel.scapegoat.inspections
+
+import com.sksamuel.scapegoat.*
+import dotty.tools.dotc.ast.Trees.*
+import dotty.tools.dotc.ast.tpd
+import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Symbols.Symbol
+import dotty.tools.dotc.core.Types.TermRef
+import dotty.tools.dotc.util.SourcePosition
+
+class AvoidRequire
+  extends Inspection(
+    text = "Use of require",
+    defaultLevel = Levels.Warning,
+    description = "Use require in code.",
+    explanation = "Using require throws an untyped Exception."
+  ) {
+
+  import tpd.*
+
+  def inspect(feedback: Feedback[SourcePosition], tree: tpd.Tree)(using ctx: Context): Unit = {
+    val traverser = new InspectionTraverser {
+      def traverse(tree: Tree)(using Context): Unit = {
+        tree match {
+          case f@Apply(ident: Ident, _) if ident.name.toString == "require" =>
```

Comment:
It seems `f` is unused, isn't it?
```suggestion
          case Apply(ident: Ident, _) if ident.name.toString == "require" =>
```

Reply:
right. Removed unused `f`